.gitignore

```diff
@@ -91,3 +91,4 @@ $RECYCLE.BIN/
 # End of https://www.gitignore.io/api/macos,windows,linux
 
 .idea
+*.db
```

Comment:
Click on the file name to see it. Looks like the review comment hid the red mark